db-migrations/1_schema_creation.up.sql

```diff
@@ -16,7 +16,7 @@ CREATE TYPE endpoint_status as ENUM ('pending', 'running', 'serving', 'failed',
 
 CREATE TABLE IF NOT EXISTS projects
 (
-    id                  serial PRIMARY KEY,
+    id                  bigserial PRIMARY KEY,
```

Comment:
Will it become an issue?

Reply:
that is not supposed to be here? will revert this